### DISCUSSION TITLE
Main Algorithms in this Project

### DISCUSSION THREAD

**Discussion by Agha-Muqarib** [lukas-blecher/LaTeX-OCR#87] [Q&A] [answered]
Hey Lukas, hope you're doing good. I wanted to know if there's any documentation for this project, like a report or something, if yes, can you please share it for a better understanding.

Also, since you've built this application, it must be at your fingertips. I wanted to of what are the **3 main algorithms** in this application and where do they reside in files. It'd be a great help if I can get to know this. Thanks and appreciate your work, man. Peace

**Comment by lukas-blecher** [comment, marked as answer]
Hello,
no I did not write a report for this repo. The only documentation available are a few doc strings in the code and of course the README. I'd recommend reading the related papers, linked in the references.
I've compiled a small list of where the action happens here: https://github.com/lukas-blecher/LaTeX-OCR/issues/69#issuecomment-1013846940

May I ask what you're set out to do?

**Reply by Agha-Muqarib** [reply]
Oh, I'm to understand the codebase to get the flow from start to end. For that, I wanted to know the main business logic in our current code base and that'd obviously be done by finding out what are the most important lines of code or most important algorithms on which our whole application stands. I'll calculate their complexities then for the best and worst case.
